src/registrationtools/spatial_registration.py

```diff
@@ -553,11 +562,12 @@ def compute_trsfs(self, p: trsf_parameters):
                     + res_trsf
                     +  # ' -res-voxel-trsf ' + res_voxel_trsf + \
                     # ' -res ' + flo_out +\
-                    " -composition-with-initial",
+                    #" -composition-with-initial"
+                    th, 
                     shell=True,
                 )
                 call(
-                    p.path_to_bin + "invTrsf %s %s" % (res_trsf, res_inv_trsf),
+                    p.path_to_bin + "invTrsf %s %s" % (res_trsf, res_inv_trsf) +th,
```

Comment:
Why do you need to add the threshold part? I would think that it is only necessary to compute the transformation.